projects/packages/status/changelog/remove-status-host_guess_from_dns

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Removing public functions should probably be a major version bump.

If we keep them uncalled and deprecated, that could be a minor bump instead (and might be more friendly to any third parties that actually use it).

Reply:
@robertsreberski thoughts? Should I restore some of this?

Reply:
It's unfortunate it was defined as a public function. My hunch is that it was only utilized by My Jetpack, but an idea of the stub might be a safe go-to here.

Reply:
Following up here, we went ahead and made this a major release out of an abundance of caution.